frontend/templates/product/MetaTags.tsx

```diff
@@ -135,7 +135,9 @@ export function MetaTags({ product, selectedVariant }: MetaTagsProps) {
                name: metaTitle || undefined,
                url: selectedVariantUrl,
                aggregateRating:
-                  product.rating && product.reviewsCount
+                  product.rating?.value &&
+                  product.reviewsCount &&
+                  (product.rating.value >= 4 || product.reviewsCount > 10)
```

Comment:
The spec says "If the average Rating is <4 and there are <10 total reviews, we should not show the Reviews section", so I would expect this to be

```suggestion
                  (product.rating.value >= 4 || product.reviewsCount >= 10)
```

But not sure if @jeffsnyder cares about that edge case.

Reply:
Yep I know this is a slight deviation but there were already other references in code that enforced `> 10` without the equal.

I also checked the old repo and you can see [here](https://github.com/iFixit/ifixit/blob/5dc8b9d17fefc794a2e43b58746f0bd367922289/Cart/Objects/ProductReviews.php#L14-L15) that `10` is the max values of reviews **to hide**. 
That's why it was coded as above

Reply:
Thanks for the explanation!